src/Classes/Board.js

```diff
@@ -122,9 +161,39 @@ export default class Board {
             organism.y = checkOrganismCoordinates.newY;
             currentTile.currentOrganism = null;
             newTile.currentOrganism = organism;
+            this.rotateOrganism(checkOrganismCoordinates.newX, currentX, checkOrganismCoordinates.newY, currentY, organism);
         }
     };
 
+    rotateOrganism = (currentX, newX, currentY, newY, organism) => {
+        const icon = organism.icon;
+        const howXChanged = currentX - newX;
+        const howYChanged = currentY - newY;
+        if (howXChanged === -1 && howYChanged === 0){
```

Comment:
Having many `if...else` statements doesn't look very good. We could improve it by creating a separate function.

```
getOrganismRotation(howXChanged, howYChanged) {
  if (howXChanged === -1 && howYChanged === 0) {
    return 'rotate(-0.5turn)';
  }
  if (howXChanged === 0 && howYChanged <= -1) {
    return 'rotate(0.25turn)';
  }
  // ...
}

rotateOrganism = (currentX, newX, currentY, newY, organism) => {
  const icon = organism.icon;
  const howXChanged = currentX - newX;
  const howYChanged = currentY - newY;

  const rotation = this.getOrganismRotation(howXChanged, howYChanged);

  icon.style.transform = 'rotate(0.25turn)';
}
```